app/views/conferences/_form_modal.html.slim

```diff
@@ -0,0 +1,32 @@
+.modal.fade data-modal='new-conference-modal' tabindex='-1' role='dialog'
+  .modal-dialog role='document'
+    .modal-content
+      = simple_nested_form_for Conference.new, data: { form: 'new-conference-form', type: "json" }, remote: true do |f|
+        .modal-header
+          h2.modal-title Create a new conference
+          button.close type='button' data-dismiss='modal'
+            span &times;
+
+        .modal-body
+          .alert.alert-danger.hidden data-errors='new-conference-modal-errors'
+
+          p
+            h3 Conference info
+          fieldset
+           = f.input :name, required: true, placeholder: 'Conference name', wrapper_html: { class: 'col-md-12' }
+           = f.input :url, required: true, placeholder: 'Conference website', wrapper_html: { class: 'col-md-12' }
+           = f.input :twitter, placeholder: '@', wrapper_html: { class: 'col-md-12' }
+           = f.input :starts_on, as: :date, html5: true, required: true, label: 'Conference start date', wrapper_html: { class: 'col-md-6' }
+           = f.input :ends_on, as: :date, html5: true, required: true, label: 'Conference end date', wrapper_html: { class: 'col-md-6' }
+           = f.input :notes, label: 'Conference notes', wrapper_html: { class: 'col-md-12' }
+
+          p
+           h3 Conference location
+          fieldset
+            = f.input :country, required: true, as: :country, prompt: "Conference country", wrapper_html: { class: 'col-md-12' }
+            = f.input :region, required: true, collection: Conference::REGION_LIST, prompt: 'Conference region', wrapper_html: { class: 'col-md-12' }
+            = f.input :city, required: true, placeholder: 'Conference city', wrapper_html: { class: 'col-md-12' }
+
+        .modal-footer
+          button.btn.btn-default type='button' data-dismiss='modal' Cancel
+          = f.submit 'Save', class: 'btn btn-success'
```

Comment:
I did not yet try this in the browser, but is it possible to "DOS-click" the button and submit the form a million times? If so, it would be cool to disable the button after submit ✌️